crates/viewer/re_renderer/src/video/player.rs

```diff
@@ -223,8 +225,7 @@ impl VideoPlayer {
             //
             // By resetting the current GOP/sample indices, the frame enqueued code below
             // is forced to reset the decoder.
-            self.current_gop_idx = usize::MAX;
-            self.current_sample_idx = usize::MAX;
+            self.last_requested_gop_idx = usize::MAX;
```

Comment:
this is a bit wonky imho - why not just `self.reset` directly. A: This would lead to a double reset.
In the interest of keeping the changes here more minimal I'm not touching this piece